### PR TITLE
[10994] Add PropertyPolicyQos Options section

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/816/)

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/818/)

File: docs/fastdds/property_policies/logging.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+
+.. _property_policies_logging:
+
+Logging Module Settings
+-----------------------
+
+.. note::
```
Comment (EduPonz) [inline docs/fastdds/property_policies/logging.rst:9]: I'd promote these to warnings

File: docs/fastdds/property_policies/statistics.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.. include:: ../../03-exports/aliases.include
+.. include:: ../../03-exports/aliases-api.include
+
+.. _property_policies_statistics:
+
+Statistics Module Settings
```
Comment (EduPonz) [inline docs/fastdds/property_policies/statistics.rst:6]: I'd say this is a non-consolidated QoS that we hope to promote to extension one day

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/823/)

**Review by EduPonz — APPROVED** [review]
LGTM

**Pull request merged**